torch/distributed/utils.py

```diff
@@ -2,7 +2,10 @@
 
 import torch
 import torch.distributed as dist
-from torch.nn.parallel._functions import _get_stream
+import importlib
+import types
```

Comment:
Why those imports are needed here?

Reply:
> Why those imports are needed here?

Oh, sorry, I forgot to remove them from last version. I will remove that. Thanks for the reminder.